esx_service/utils/auth_data.py

```diff
@@ -0,0 +1,306 @@
+# VM based authorization for docker volumes
+
+import sqlite3
+import uuid
+import os
+
+class DbConnectionError(Exception):
+    """ An exception thrown when connection to a sqlite database fails """
+
+    def __init__(self, db_path):
+        self.db_path = db_path
+
+    def __str__(self):
+        return "DB connection error %s" % self.db_path
+
+class DockerVolumeTenant:
+
+    def __init__(self, name, description, default_datastore, default_privileges,
+                    vms, privileges, id=None):
+            self.name = name
+            self.description = description
+            self.default_datastore = default_datastore
+            self.default_privileges = default_privileges
+            self.vms = vms
+            self.privileges = privileges
+            if not id:
+                self.id = str(uuid.uuid4())
+            else:
+                self.id = id
+        
+    def add_vm_to_tenant(self, conn, vm_id, vm_name):
+        tenant_id = self.id
```

Comment:
I don't disagree with that but it does looks like useless op. I would have named the member var tenant_id (not id) and then it would have been self.tenant_id. 
At any rate, this is a minor taste preference, feel free to keep as is.